intermediate-python-debugging/problem-set-using-the-debugger.md

```diff
@@ -2,7 +2,7 @@
 
 ## Directions
 
-In this problem set we will use the debugger to identify two errors.  To run the activity clone the code at [AdaGold/python-debugging](https://github.com/AdaGold/python-debugging).  Then follow the directions in the README to set up the repo.
+In this problem set we will use the debugger to identify two errors.  To run the activity clone the code at [AdaGold/python-debugging](https://github.com/AdaGold/python-debugging/tree/cm/dictionary-version).  Then follow the directions in the README to set up the repo.
```

Comment:
Rather than pointing at the branch, we should merge the branch into main in AdaGold. Students haven't seen how to fetch a branch from a repo.

Could you turn the branch into a PR?

From looking at the branch, so far I would say:
- remove .pylint from the repo
- remove __init__.pyc from the repo (I've never seen this, usually byte code gets put in __pycache__ dirs)
- add enough blanks (I think just one) to the top of order.py so that the line numbers line up with the  error text in this lesson (which will also require updating running-debugger.png)--there's another screenshot that shows the breakpoint still on line 16.

Reply:
Could you also replace `__init__` on line 79 with `create_product`
And as long as you're in this file, could you replace `Debug Test` on line 106 with <code>Debug Test</code>
It doesn't look like learn is interpreting the markdown in the summary tag properly.